### ISSUE TITLE
Следует добавить  терминал возможность запускать игры

### ISSUE THREAD

**Issue by Nikita-bunikido** [extremecodetv/ExtremeCodeOS#106]
В папке Games лежат игры для ExtremeCodeOS.
Предлагаю сделать так, чтобы игры можно было
запускать из терминала (kernel) к примеру:
``` game <gamename>```
Или что-то тип того
Кто что думает по этому?

**Comment by yosa12978** [comment]
Это нарушает концепцию - никаких терминалов

**Comment by ketsushiri** [comment]
> Это нарушает концепцию - никаких терминалов
Запилить большую кнопку "Запустить игру". Бремя выбора игры для запуска будет ложиться на операционную систему.

**Comment by Nikita-bunikido** [comment]
> Это нарушает концепцию - никаких терминалов

Да, нарушает но терминал уже находится в корне: ``` ExtremeCodeOS\src\Kernel\kernel.c```
Я значительно переработал его архитектуру и подумал над добавлением команды.
Тем более терминалом будут пользоваться дофига прошаренные администраторы, 
а не домохозяйки. Об этом вроде в ролике говорилось

**Comment by invizex** [comment]
Можно добавить команду которая выводить инфу о **ОС** (типа **neofetch**)

**Comment by Nikita-bunikido** [comment]
> Можно добавить команду которая выводить инфу о **ОС** (типа **neofetch**)

Так и сделаем
